app/commands/all_commands.js

```diff
@@ -46,12 +46,41 @@ const DICE_COMMAND = {
     ]
 }
 
-const ALL_COMMANDS = [TEST_COMMAND, QUOTE_COMMAND, DICE_COMMAND];
+const LOCALISED_TIME_COMMAND = {
+    name: 'localised_time',
+    description: 'Provides a localised time',
+    type: 1,
+    integration_types: [0, 1],
+    contexts: [0, 1, 2],
+    options: [
+        {
+            name: 'date',
+            type: 3, // String
+            description: 'Enter the date in `yyyy-MM-dd` format',
```

Comment:
(suggestion ) instead of making this required, it could take today as the date if no date is given.